nuxeo-features/nuxeo-platform-csv-export/src/test/java/org/nuxeo/ecm/platform/csv/export/TestCSVExportAction.java

```diff
@@ -338,8 +331,15 @@ public void write(int b) throws IOException {
         }
 
         List<String> lines = Files.lines(testCsv.toPath()).collect(Collectors.toList());
+        int headerCount = 0;
+        for (String line : lines) {
+            if (line.startsWith("repository,uid")) {
+                headerCount++;
+            }
+        }
```

Comment:
Could be replaced with : 
```suggestion
        long count = lines.stream().filter(l -> l.startsWith("repository,uid")).count();
```